src/Microsoft.ML.Entrypoints/Microsoft.ML.Entrypoints.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Ya, I guess the options are to include it in the main nuget as it was in the Legacy project, or to create a separate nuget for projects that want to use entrypoints.  @yaeldekel if we went with the latter, would it simply work out of the box?

Reply:
Good point. It should probably be part of the Microsoft.ML nuget? Or should we have a separate nuget for it?

---
In reply to: [240758736](https://github.com/dotnet/machinelearning/pull/1859#discussion_r240758736) [](ancestors = 240758736)

Reply:
I believe @eerhardt mentioned that he thought entrypoints could go in their own nuget.  I'm not sure what the tradeoffs would be though.

---
In reply to: [240798793](https://github.com/dotnet/machinelearning/pull/1859#discussion_r240798793) [](ancestors = 240798793,240758736)

Reply:
I think we should have a separate NuGet because most of our consumers (normal .NET developers) won't ever need this functionality.

People can opt-in to this functionality by referencing the `EntryPoint` nuget package.